openedx/core/djangoapps/credit/email_utils.py

```diff
@@ -180,3 +206,60 @@ def _email_url_parser(url_name, extra_param=None):
     dashboard_url_path = reverse(url_name) + extra_param if extra_param else reverse(url_name)
     dashboard_link_parts = ("https", site_name, dashboard_url_path, '', '', '')
     return urlparse.urlunparse(dashboard_link_parts)
+
+
+def get_credit_providers_by_course(user, course_key):
+    """Get the course information from ecommerce and parse the data to get providers.
+
+    Arguments:
+        user (User): The user object.
+        course_key (CourseKey): The identifier for the course.
+
+    Returns:
+        list, containing course providers.
+    """
+    credit_config = CreditConfig.current()
+
+    # Bypass caching for staff users, who may be adding providers and want
+    # to see them immediately.
```

Comment:
This comment isn't accurate. Please remove it. You're doing no such bypassing of the cache here.